app/src/renderer/components/common/AppHeader.vue

```diff
@@ -66,7 +66,7 @@ export default {
     position fixed
     top 0
     left 0
-    z-index 100
+    z-index 2000
```

Comment:
1000 is the z-index of modals - I'm going to revert this one because I don't think the app header should show on top of modals